{"name": "test_get_order_by_id", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 200\n +  where 404 = <Response [404]>.status_code", "trace": "def test_get_order_by_id():\n        \"\"\"\n        finds order by id\n        :return:\n        \"\"\"\n        logging.info(\"finding order by id\")\n        response = requests.get(f\"{url}/store/order/1\", headers=header)\n>       assert response.status_code == 200\nE       assert 404 == 200\nE        +  where 404 = <Response [404]>.status_code\n\ntest.py:128: AssertionError"}, "description": "\n    finds order by id\n    :return:\n    ", "start": 1659615839033, "stop": 1659615839633, "uuid": "814a7487-ec2c-44fc-8bc4-9c87e1003e99", "historyId": "0b584f1df0a625daba0cc5e2748d9176", "testCaseId": "a72e6b400c08808a459eba197367a0fe", "fullName": "test#test_get_order_by_id", "labels": [{"name": "suite", "value": "test"}, {"name": "host", "value": "DESKTOP-V80V9RG"}, {"name": "thread", "value": "22264-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test"}]}